src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/NonStandardDurationReason.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.sscs.reference.data.mappings;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum NonStandardDurationReason {
+
+    TIME_FOR_INTERPRETATION("timeForInterpretation", "Time for interpretation", "", ""),
+    COMPLEX_CASE("complexCase", "Complex Case", "", ""),
+    LITIGANT_IN_PERSON("litigantInPerson", "Litigant in person", "", "");
+
+    private final String hmcReference;
+    private final String valueEn;
+    private final String valueCy;
+    private final String serviceCode;
```

Comment:
Where are the service codes?

Reply:
Non provided at the moment.

Reply:
Our service code will always be BBA-3, see the other comments. You should keep consistency re nulls or empty strings